lib/tasks/jenkins.rake

```diff
@@ -11,10 +10,10 @@ begin
 
     namespace :setup do
       task :pre_ci do
-        ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
-        gem 'ci_reporter'
+        require 'minitest/ci'
```

Comment:
```suggestion
        ENV['USE_MEAN_TIME_REPORTER'] == '0'
        require 'minitest/ci'
```

This might be what makes Katello confuse the shit out of `Minitest::Ci`:

https://github.com/Katello/katello/blob/30e6fe7d91d6296bb02830e4afaa3f66bee38795/test/katello_test_helper.rb#L12-L15